### ISSUE TITLE
Dependency bug fails Docker image build

### ISSUE THREAD

**Issue by AntoineDao** [ladybug-tools/honeybee-energy#563]
It seems there is a dependency management issue when building the Docker image: https://github.com/ladybug-tools/honeybee-energy/runs/1767450994#step:7:32901

Is blocking: https://github.com/pollination/private-garden/issues/42

**Comment by mostaphaRoudsari** [comment]
This is a strange one honeybee-core==1.42.0 is installed and right after that there is a call for honeybee-core[cli]==1.41.12 and that makes things fail.

![image](https://user-images.githubusercontent.com/2915573/106006892-b5158200-6083-11eb-8308-15e6261cec76.png)

**Comment by AntoineDao** [comment]
The plot thickens... I just tried building the Docker image locally and it runs fine... Weird. I wonder if there is some sort of weird caching going on.

**Comment by mostaphaRoudsari** [comment]
It's very possible since the version 1.41.12 is the one right before 1.42.0 - maybe we should just try to push a fix and create a new release and see if that works.

**Comment by AntoineDao** [comment]
Is it ok if I manually push the right version of the image to dockerhub for now so we don't have to update the honeybee-energy Queenbee plugin?

**Comment by mostaphaRoudsari** [comment]
I was just about to write that! :D Let's do that for now so we can move forward.

**Comment by AntoineDao** [comment]
Ok so, I did a bit more digging and it seems that this commit fixes the image build: d17eb5f801ce25ea2dcadd243cc0667d925f9ef7

I tried building the Docker image with the commits that previously failed and they also consistently failed. I think we need to find away to ensure that dependencies that are repeated twice due to CLI requirements are in sync. In this case it seems that `honeybee-core` is set in both `requirements.txt` and in the `extras_require` of `setup.py`.
